pages/projects-list/index.tsx

```diff
@@ -0,0 +1,202 @@
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const debouncedSetQuerySearch = useCallback(
+    debounce((val) => {
+      setQuerySearch(val);
+    }, 500),
+    []
+  );
+
+  const handleSearchInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setSearchTextInput(e.target.value);
+    debouncedSetQuerySearch(e.target.value);
+    setPage(0);
+  };
+
+  const handleCohortYearChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setSelectedCohortYear(Number(e.target.value) as Cohort["academicYear"]);
+    setPage(0);
+  };
+
+  /** To fetch more projects when the bottom of the page is reached */
+  const observer = useRef<IntersectionObserver | null>(null);
+  const bottomOfPageRef = createBottomOfPageRef(
+    isFetching(fetchProjectsStatus),
+    hasMore,
+    setPage,
+    observer
+  );
+
+  useEffect(() => {
+    if (currentCohortYear) {
+      setSelectedCohortYear(currentCohortYear);
+    }
+  }, [currentCohortYear]);
+
+  return (
+    <Body isLoading={isLoadingCohorts}>
+      <Stack direction="column" mt="0.5rem" mb="1rem">
+        <Stack
+          direction="row"
+          justifyContent="space-between"
+          alignItems="center"
+          width="100%"
+          mb={{ md: "0.5rem" }}
+        >
+          <TextField
+            label="Search"
+            value={searchTextInput}
+            onChange={handleSearchInputChange}
+            size="small"
+          />
+          <TextField
+            name="cohort"
+            label="Cohort"
+            value={selectedCohortYear}
+            onChange={handleCohortYearChange}
+            select
+            size="small"
+          >
+            {cohorts &&
+              cohorts.map(({ academicYear }) => (
+                <MenuItem key={academicYear} value={academicYear}>
+                  {academicYear}
+                </MenuItem>
+              ))}
+          </TextField>
+        </Stack>
+
+        <Tabs
+          value={selectedLevelOfAchievement}
+          onChange={handleTabChange}
+          textColor="secondary"
+          indicatorColor="secondary"
+          aria-label="project-level-tabs"
+          variant="scrollable"
+          scrollButtons="auto"
+          allowScrollButtonsMobile
+          sx={{
+            [`& .${tabsClasses.scrollButtons}`]: { color: "primary" },
+            marginY: { xs: 2, md: 0 },
+          }}
+        >
+          {Object.values(LEVELS_OF_ACHIEVEMENT_WITH_ALL).map((level) => {
+            return <Tab key={level} value={level} label={level} />;
+          })}
+        </Tabs>
+      </Stack>
+      <LoadingWrapper
+        isLoading={
+          (projects === undefined || projects.length === 0) &&
+          isFetching(fetchProjectsStatus)
+        }
+      >
+        <NoDataWrapper
+          noDataCondition={projects === undefined || projects.length === 0}
+          fallback={<NoneFound message="No such projects found" />}
+        >
+          <ProjectTable projects={projects} mutate={mutate}>
```

Comment:
I think this is a typo? The `ProjectTable` component should not have any children